playground/package.json

```diff
@@ -28,7 +28,7 @@
     "react-scripts": "3.1.2"
   },
   "scripts": {
-    "start": "react-app-rewired start",
+    "start": "react-app-rewired --openssl-legacy-provider start",
```

Comment:
please revert, as with this I cannot start local playground anymore, and need to manually remove that flag